client/lib/signup/hint-data.js

```diff
@@ -1,673 +1,673 @@
+	'Design portfolio': i18n.translate( 'Design portfolio' ),
+	Dessert: i18n.translate( 'Dessert' ),
+	Devops: i18n.translate( 'Devops' ),
+	'Diagnostic machine design': i18n.translate( 'Diagnostic machine design' ),
+	Diary: i18n.translate( 'Diary' ),
+	'Digital design': i18n.translate( 'Digital design' ),
+	'Digital portfolio': i18n.translate( 'Digital portfolio' ),
+	Digitization: i18n.translate( 'Digitization' ),
+	Disease: i18n.translate( 'Disease' ),
+	'Dog boarding kennel': i18n.translate( 'Dog boarding kennel' ),
+	'Dog grooming': i18n.translate( 'Dog grooming' ),
+	Dogs: i18n.translate( 'Dogs' ),
+	'Domestic violence prevention': i18n.translate( 'Domestic violence prevention' ),
+	'Drawing portfolio': i18n.translate( 'Drawing portfolio' ),
+	Drawings: i18n.translate( 'Drawings' ),
+	'Dream coaching': i18n.translate( 'Dream coaching' ),
+	'Dream interpretations': i18n.translate( 'Dream interpretations' ),
+	Dreams: i18n.translate( 'Dreams' ),
+	Dresses: i18n.translate( 'Dresses' ),
+	Drinks: i18n.translate( 'Drinks' ),
+	Drugs: i18n.translate( 'Drugs' ),
+	'Dungeons and dragons': i18n.translate( 'Dungeons and dragons' ),
+	'E-book': i18n.translate( 'E-book' ),
+	'E-books': i18n.translate( 'E-books' ),
+	'E-commerce': i18n.translate( 'E-commerce' ),
+	'Early childhood services': i18n.translate( 'Early childhood services' ),
+	Ecological: i18n.translate( 'Ecological' ),
+	Ecosystem: i18n.translate( 'Ecosystem' ),
+	Editing: i18n.translate( 'Editing' ),
+	Education: i18n.translate( 'Education' ),
+	'Educational books': i18n.translate( 'Educational books' ),
+	'Educational services': i18n.translate( 'Educational services' ),
+	Eggs: i18n.translate( 'Eggs' ),
+	'Electrical design': i18n.translate( 'Electrical design' ),
+	'Electrical panels': i18n.translate( 'Electrical panels' ),
+	Electrician: i18n.translate( 'Electrician' ),
+	'Electronic components': i18n.translate( 'Electronic components' ),
+	'Electronic music': i18n.translate( 'Electronic music' ),
+	Electronics: i18n.translate( 'Electronics' ),
+	Embroidery: i18n.translate( 'Embroidery' ),
+	Emotions: i18n.translate( 'Emotions' ),
+	Employment: i18n.translate( 'Employment' ),
+	'Energy efficient appliances': i18n.translate( 'Energy efficient appliances' ),
+	Engineering: i18n.translate( 'Engineering' ),
+	'Engineering services': i18n.translate( 'Engineering services' ),
+	'Engineering software': i18n.translate( 'Engineering software' ),
+	English: i18n.translate( 'English' ),
+	'English lessons': i18n.translate( 'English lessons' ),
+	Enterprise: i18n.translate( 'Enterprise' ),
+	Entertainment: i18n.translate( 'Entertainment' ),
+	Entrepreneur: i18n.translate( 'Entrepreneur' ),
+	Entrepreneurship: i18n.translate( 'Entrepreneurship' ),
+	Environment: i18n.translate( 'Environment' ),
+	'Environment consultant': i18n.translate( 'Environment consultant' ),
+	Equestrian: i18n.translate( 'Equestrian' ),
+	Equity: i18n.translate( 'Equity' ),
+	Evangelism: i18n.translate( 'Evangelism' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Evangelisim' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).